AndroidTest/app/build.gradle

```diff
@@ -103,6 +108,24 @@ task(uploadFixtures,type:Exec) {
     commandLine "$android/platform-tools/adb","push", "-p", "$scriptLocation/../../fixture", "/sdcard/fixture"
 }
 
+def convertTestSysPropsToHash (){
```

Comment:
Here is a slightly more 'groovy' (!) way to do it (I'm not too sure on my indenting):
```
def convertTestSysPropsToHash2 (){
    return "{" +
        System.properties.grep { prop ->
        prop.key.startsWith("test") }
        .collect(new ArrayList()) { prop -> "{\""+prop.key+"\",\""+ prop.value+"\"}"}
        .join(",") +
        "}"
}
```